### ISSUE TITLE
schema.json missing in 4.x line?

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [webpro/knip#455]
`https://unpkg.com/knip@latest/schema.json` used to work as a `"$schema"` property in `knip.json` files. But now it's a 404.

https://unpkg.com/browse/knip@4.0.1/schema-jsonc.json does exist, but:

```json
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "title": "Knip configuration for JSONC",
  "description": "See https://github.com/webpro/knip",
  "allOf": [
    {
      "$ref": "https://unpkg.com/knip@4/schema.json"
    }
  ],
  "allowTrailingCommas": true
}
```

...and that `https://unpkg.com/knip@4/schema.json` is the same 404.

Is this a regression?

**Comment by webpro** [comment]
Dang, this was definitely a regression 😭 Thanks for reporting, should be back by now :)

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v4.0.2. See [Release 4.0.2](https://github.com/webpro/knip/releases/tag/4.0.2) for release notes._
